pulseapi/entries/views.py

```diff
@@ -89,14 +96,16 @@ class EntriesListView(ListCreateAPIView):
 
     **Route** - `/entries`
 
-    **Query Parameters** -
+    #Query Parameters -
 
     - `?search=` - Allows search terms
     - `?tag=` - Allows filtering entries by a specific tag
     - `?issue=` - Allows filtering entries by a specific issue
+    - `?page=` - Page number, default size of 50
```

Comment:
the 'default...' information seems to be on the wrong field here. `Page number, limits returned results to [page*page_size, (page+1)*page_size)`?